opentelemetry-extra/exe/ot-write-benchmark/Main.hs

```diff
@@ -0,0 +1,48 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Main where
+
+import Criterion.Main
```

Comment:
Please use `gauge` instead of `criterion` unless you really want some criterion-specific features.

Reply:
fix